src/main/java/com/google/alpollo/TextToSpeechServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.alpollo;
+
+import com.google.alpollo.model.Lyrics;
+import com.google.gson.Gson;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/text-to-speech")
+public class TextToSpeechServlet extends HttpServlet {
+  private final Gson gson = new Gson();
+
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    try {
+      String lyrics = gson.fromJson(request.getReader(), Lyrics.class).getLyrics();
+      String projectID = ConfigHelper.getSensitiveData(this.getServletContext(), ConfigHelper.SENSITIVE_DATA.PROJECT_ID);
+      ByteString audioContents = TextToSpeechService.convertTextToSpeech(lyrics, projectID);
+
+      if (audioContents == null) {
+        response.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR,
+            "Couldn't generate audio from the text.");
+        return;
+      }
+
+      byte[] audio = audioContents.toByteArray();
+      response.setContentType("application/json;");
+      response.getWriter().println(new Gson().toJson(audio));
+    } catch (Exception e) {
```

Comment:
Is it ok to catch all exceptions instead of specifying each type of exception?

Reply:
Yes, since we should not throw any exceptions from the servlet

Reply:
Well not throw them, but specifiy which exceptions you're trying to catch.